yt/yt/server/job_proxy/user_job.cpp

```diff
@@ -1178,40 +1179,70 @@ class TUserJob
         YT_LOG_INFO("Pipes initialized");
     }
 
+    void SetEnvironment(const TString& variable)
```

Comment:
Let's overload this method and do not serialize/deserialize strings

Reply:
What do you mean? I don't see reason to do any more complicated optimizations here. It's a few kb at most.

Reply:
I mean that it would be better to implement 2 more variable setters than Format(...) strings just to pass arguments to the function.

Yes, it's cold code, but I'm worried about style.

Reply:
To be honest, I'm not sure that this is _that_ important. From my perspective, function with two arguments is a little bit nicer, but it's really a small thing.